data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1704 @@
+				`	"House Bebliss, right? Thank you for finally finding the time."`
+					goto remember
+			`	"To our employee, what you said, it is," they answer. "To 'come in person next time' you requested."`
+			label remember
+			`	They 'invite' you to follow you into what looks like an expensive restaurant, serving typical Arach cuisine. The bulky Arachi don't give you much of a choice, barring you from going elsewhere, only letting up when you are seated. The two Arachi businessmen say they will pay for your meal, though you can't quite find much in the menu appealing to the human palate.`
+			`	"Usually more... respectful about such requests from our House, people are, but hasty of us to approach a human Captain like we would an Arach one, it may have been," one of them says, going through the menu. "Still interested, the House is, so to entertain your request, we came."`
+			`	"Interested in your assistance, that is," the other continues, as he signals for one of the bodyguards to get a waiter. "An important job for you, we have."`
+			choice
+				`	"Alright, you have my interest. What's this job about?"`
+				`	"Well if you want me to work with you so much, why don't you just tell me what the job is?"`
+				`	"I'm sorry, but I'm not looking for any work now."`
+					decline
+				`	"Second chance, tenth chance, I really don't care how many you give me. I'm not interested."`
+					decline
+			label accept
+			`	"Say much, here we cannot," he answers. "If interested, you are, to <destination>, you must go. To our House's private estate there."`
+			`	"That it concerns recent human events, what we can say is. Events that involved in, you were," the other says, handing you some coordinates to their estate on <planet>.`
+			`	The waiter arrives, and they suggest some dishes, seeing as you didn't pick anything from the menu. The smell coming from the other tables, however, tells you to decline, so you politely leave the establishment. It seems the only way to find out what these Arachi want is to head to <destination>.`
+				accept
+
+mission "House Bebliss 2-FW"
+	name "Old Friends"
+	description "Go to <destination> and look for Freya to see if she is willing to let the members of House Bebliss study the Pug artifacts with her."
+	landing
+	to offer
+		not "Heliarch License: done"
+		not "Heliarch Investigation 2: active"
+		or
+			has "House Bebliss 1-A: done"
+			has "House Bebliss 1-B: done"
+	source "Factory of Eblumab"
+	destination "Pugglemug"
+	on offer
+		conversation
+			`You follow the coordinates given to you and land on top of a mountain, on a set of private hangars close to an estate covered by a dome similar to the one over the spaceport city. You are received by who you assume are Arachi butlers and maids, given their tidy, well-kept uniforms, and follow one of them into the mansion.`
+			`	They bring you to a large, wide room where three preoccupied Arachi are pouring over schematics and blueprints scattered across a table.`
```

Comment:
Could you explain this one? It seems sort of redundant to me, since the player is already told there are three Arachi there, they see the three Arachi and all, so it's not like they're shouting to annouce their presence.
In the next paragraph we have the greeting portion right away, so it's not something introducing someone's dialog where there was nothing of the sort before either.

Reply:
Now that I think of it, you're right.